core/src/com/itemshop/factories/characters/PlayerFactory.java

```diff
@@ -42,6 +43,9 @@ public static Entity create() {
 		};
 		player.add(walkingComponent);
 		
+		// All characters will have a render offset so they are drawn in the centre of a tile.
+		player.add(new RenderOffsetComponent(0f, 0.25f));
```

Comment:
Assuming we want to keep the texture always aligned to the 16x16 grid, it might be better to be passing "4 pixels" rather than "1/4 of a texture", could keep it as ints that way